### ISSUE TITLE
 Looking for peers

### ISSUE THREAD

**Issue by Parmanand0712** [base/node#319]
INFO [09-09|17:04:48.810] Looking for peers                        peercount=0 tried=114 static=0
INFO [09-09|17:04:58.830] Looking for peers                        peercount=0 tried=153 static=0
INFO [09-09|17:05:09.322] Looking for peers                        peercount=0 tried=96  static=0
INFO [09-09|17:05:19.343] Looking for peers                        peercount=1 tried=51  static=0
INFO [09-09|17:05:29.814] Looking for peers                        peercount=0 tried=173 static=0
INFO [09-09|17:05:39.818] Looking for peers                        peercount=1 tried=104 static=0
INFO [09-09|17:05:49.866] Looking for peers                        peercount=1 tried=146 static=0
INFO [09-09|17:05:59.983] Looking for peers                        peercount=2 tried=90  static=0
INFO [09-09|17:06:10.018] Looking for peers                        peercount=1 tried=167 static=0
INFO [09-09|17:06:20.175] Looking for peers                        peercount=1 tried=89  static=0
INFO [09-09|17:06:30.274] Looking for peers                        peercount=1 tried=106 static=0
INFO [09-09|17:06:40.472] Looking for peers                        peercount=1 tried=102 static=0
INFO [09-09|17:06:50.536] Looking for peers                        peercount=1 tried=175 static=0
INFO [09-09|17:07:01.218] Looking for peers                        peercount=1 tried=135 static=0
INFO [09-09|17:07:11.229] Looking for peers                        peercount=1 tried=117 static=0
INFO [09-09|17:07:21.315] Looking for peers                        peercount=1 tried=120 static=0
INFO [09-09|17:07:31.320] Looking for peers                        peercount=1 tried=96  static=0
INFO [09-09|17:07:41.419] Looking for peers                        peercount=1 tried=85  static=0
INFO [09-09|17:07:51.452] Looking for peers                        peercount=1 tried=163 static=0
INFO [09-09|17:08:01.518] Looking for peers                        peercount=1 tried=95  static=0
INFO [09-09|17:08:11.577] Looking for peers                        peercount=1 tried=94  static=0
INFO [09-09|17:08:21.712] Looking for peers                        peercount=1 tried=105 static=0
INFO [09-09|17:08:31.713] Looking for peers                        peercount=1 tried=170 static=0
INFO [09-09|17:08:41.718] Looking for peers                        peercount=1 tried=114 static=0
INFO [09-09|17:08:52.006] Looking for peers                        peercount=1 tried=161 static=0
INFO [09-09|17:09:02.023] Looking for peers                        peercount=2 tried=131 static=0
INFO [09-09|17:09:22.107] Looking for peers                        peercount=2 tried=139 static=0
INFO [09-09|17:09:32.570] Looking for peers                        peercount=2 tried=105 static=0
INFO [09-09|17:09:42.707] Looking for peers                        peercount=2 tried=110 static=0
INFO [09-09|17:09:52.801] Looking for peers                        peercount=2 tried=132 static=0
INFO [09-09|17:10:02.851] Looking for peers                        peercount=2 tried=164 static=0
INFO [09-09|17:10:12.944] Looking for peers                        peercount=2 tried=72  static=0
INFO [09-09|23:53:37.332] New local node record                    seq=1,699,529,870,615 id=dc0b60b53f8b847b ip=5.199.165.40 udp=30303 tcp=30303
INFO [09-09|23:53:39.022] New local node record                    seq=1,699,529,870,616 id=dc0b60b53f8b847b ip=5.199.165.40 udp=45278 tcp=30303
INFO [09-09|23:53:42.413] New local node record                    seq=1,699,529,870,617 id=dc0b60b53f8b847b ip=5.199.165.40 udp=30303 tcp=30303
INFO [09-10|00:01:16.858] New local node record                    seq=1,699,529,870,618 id=dc0b60b53f8b847b ip=5.199.165.40 udp=65019 tcp=30303
INFO [09-10|00:55:33.019] New local node record                    seq=1,699,529,870,619 id=dc0b60b53f8b847b ip=5.199.165.40 udp=30303 tcp=30303
INFO [09-10|01:18:20.552] New local node record                    seq=1,699,529,870,620 id=dc0b60b53f8b847b ip=5.199.165.40 udp=34713 tcp=30303
INFO [09-10|01:33:51.078] New local node record                    seq=1,699,529,870,621 id=dc0b60b53f8b847b ip=5.199.165.40 udp=30303 tcp=30303
WARN [09-10|03:57:06.963] Served eth_hashrate                      conn=162.142.125.32:59992 reqid=1 duration="20.98µs" err="the method eth_hashrate does not exist/is not available"

Is it okay, or do I need to change any values in the parameters of the docker-compose.yml file?

**Comment by SwapnilSoni1999** [comment]
Have you resolved? Because I'm facing the same issue

**Comment by Parmanand0712** [comment]
Hey Swapnil,

in my case, it was normal logs. It took a few days to sync the Base archive node without making any changes to the Docker Compose file, so just wait for some time; it should work in your case as well.

**Comment by SwapnilSoni1999** [comment]
> few days

I don't think so I found out that the data was corrupted due to low memory issue so I cleaned everything up and reran base node and it synced in 2hrs

**Comment by thanhnguyennguyen** [comment]
> > few days
> 
> I don't think so I found out that the data was corrupted due to low memory issue so I cleaned everything up and reran base node and it synced in 2hrs

did you change any configuration ?
I face same issue

**Comment by Parmanand0712** [comment]
I just downloaded the archive snapshot, set the OP_NODE_L1_ETH_RPC and OP_NODE_L1_BEACON environment variables to my L1 RPC URL in the .env file, and then started the node using Docker Compose.

**Comment by SwapnilSoni1999** [comment]
> > > few days
> > 
> > 
> > I don't think so I found out that the data was corrupted due to low memory issue so I cleaned everything up and reran base node and it synced in 2hrs
> 
> did you change any configuration ? I face same issue
> 
> and how to enable archive mode for node Base

I had to clear existing docker cache and images and prune everything then I ran and it worked
